### DISCUSSION TITLE
Android 11 support

### DISCUSSION THREAD

**Discussion by SebaUbuntu** [SebaUbuntu/TWRP-device-tree-generator#27] [Ideas]
Hi everyone

I pushed some commits to add support to Android 11, if someone will have a chance to have a native Android 11 device, let me know if it will work (due to TWRP development rate, it will take a while to see if it will work)
PRs are accepted as always ;)

**Comment by xxroot** [comment]
Work, but I encountered some problems. After compiling, I brush it to the mobile phone, and it will restart repeatedly in the twrp logo

**Reply by xxroot** [reply]
About the rules of FBE decryption, do you have the idea of adding it to the tool?

**Comment by xxroot** [comment]
@SebaUbuntu 

About the rules of FBE decryption, do you have the idea of adding it to the tool?
